src/accordionPanelGroup.test.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import AccordionPanelGroupContainer from './AccordionPanelGroupContainer';
+
+describe('AccordionPanelGroupContainer', function() {
+
+	it('exists', function() {
+		const accordionPanelGroup = TestUtils.renderIntoDocument(<AccordionPanelGroupContainer />);
+		const accordionPanelGroupNode = ReactDOM.findDOMNode(accordionPanelGroup);
+
+		expect(accordionPanelGroupNode).not.toBeNull();
+	});
+
+});
```

Comment:
a couple tests to add would be to simulate clicks on the accordion - make sure the handler gets called and that its in the expected open or closed state. Also check that the accordion panel group state gets updated.